src/bridge/bridge.c

```diff
@@ -574,6 +586,24 @@ run_bridge (const gchar *interactive,
 }
 
 static void
+print_rules (gboolean opt_privileged)
+{
+  CockpitRouter *router = NULL;
+  GList *bridges = NULL;
+  CockpitTransport *transport = cockpit_interact_transport_new (0, 1, "--");
+
+  router = setup_router (transport, opt_privileged, &bridges);
+
+  cockpit_router_dump_rules (router);
+
+  g_object_unref (router);
+  g_list_free (bridges);
+  g_object_unref (transport);
+  cockpit_packages_free (packages);
+  packages = NULL;
```

Comment:
Seems a bit odd to free a global variable here, the function neither initializes nor touches that in any way. I see that it just cleans up after that side effect of `setup_router()`. Of course, feel free to ignore this if setting it up in a "nice" way (e. g. making packages an argument of `setup_router()` and keeping it local) is too cumbersome.